include/seqan3/io/sequence_file/format_embl.hpp

```diff
@@ -37,9 +35,8 @@
 #include <seqan3/range/views/take_exactly.hpp>
 #include <seqan3/range/views/take_line.hpp>
 #include <seqan3/range/views/take_until.hpp>
-#include <seqan3/range/views/to.hpp>
+#include <seqan3/range/views/chunk.hpp>
```

Comment:
is this include even used?

what about the other includes? there seems to be a lot redundancy.

Reply:
it is not used anymore